docs/getting-started/installation.md

```diff
@@ -192,11 +197,11 @@ Or use the Prefect CLI command `prefect version`, which prints version and envir
 <div class="terminal">
 ```
 $ prefect version
-Version:             2.0
+Version:             2.3.2
```

Comment:
```suggestion
Version:             2.*
```

to make it more evergreen

Reply:
Let me think about this. There's a balance between what people would actually see and how they interpret it.